storage/aptosdb/src/schema/db_metadata/mod.rs

```diff
@@ -51,6 +51,7 @@ pub enum DbMetadataKey {
     StateSnapshotRestoreProgress(Version),
     LedgerCommitProgress,
     StateKVCommitProgress,
+    StateKvShardCommitProgress(usize),
```

Comment:
it's a backward incompatible change if you add enum variants in the middle.

Reply:
Also nit: I'd like the field to have a name or a meaningful type

Reply:
Why is it backward incompatible? (even without #[repr(u8)]) How is it serialized?

BTW does changing the enum name backward compatible? I want to change the `V` at line 53 to lower case.

Reply:
n-th variant has n wrote down as prefix, so whatever represented OverallCommitProgress is now StateKvShardCommitProgress.

renaming is fine.

Reply:
Done